Assignment/starter_code/select_fun_part2/js/main.js

```diff
@@ -1,29 +1,69 @@
 // 1.	Slide the header up when it is clicked on.
-
+$('h1').on('click', function(){
+	$('h1').slideUp();
+});
 // 2.	Remove the purplebox class
+$('#box14').removeClass('purplebox');
+
+// Do you need to target the specfic box, why does 'body' not work here?
```

Comment:
Ahh, well if you were targeting `$('body')` it would try to remove the `purplebox` class from the `<body>` element, not it's descendants. If you wanted to just find all of the elements with a `purplebox` class to remove it you could actually just target `$('purplebox')` couldn't you? Then it would just find _anything_ with a class of `purplebox`, and then you could remove it! :)